Dockerfile

```diff
@@ -29,21 +34,19 @@ RUN set -x && \
         pcre-dev \
         python \
         zlib-dev && \
+    # Nginx user & group
+    addgroup -S nginx && \
+    adduser -D -S -h /var/cache/nginx -s /sbin/nologin -G nginx nginx && \
     # Build libpng:
-    # This sadly requires an old version of http://www.libpng.org/pub/png/libpng.html
-    LIBPNG_VERSION=1.2.59 && \
     cd /tmp && \
     curl -L http://prdownloads.sourceforge.net/libpng/libpng-${LIBPNG_VERSION}.tar.gz | tar -zx && \
     cd /tmp/libpng-${LIBPNG_VERSION} && \
     ./configure --build=$CBUILD --host=$CHOST --prefix=/usr --enable-shared --with-libpng-compat && \
     make install V=0 && \
     # Build PageSpeed:
-    # Check https://github.com/pagespeed/ngx_pagespeed/releases for the latest version
-    PAGESPEED_VERSION=1.12.34.3 && \
-    NGX_PAGESPEED_VERSION=1.12.34.3 && \
     cd /tmp && \
-    curl -L https://github.com/We-Amp/ngx-pagespeed-alpine/blob/master/mod-pagespeed-beta-1.12.34.3.tar.bz2?raw=true | tar -jx && \
-    curl -L https://github.com/pagespeed/ngx_pagespeed/archive/v${NGX_PAGESPEED_VERSION}-stable.tar.gz | tar -zx && \
+    curl -L https://github.com/We-Amp/ngx-pagespeed-alpine/blob/master/mod-pagespeed-beta-${PAGESPEED_VERSION}.tar.bz2?raw=true | tar -jx && \
```

Comment:
since beta tarball renamed to stable tarball, may have to update tarball download path. Currently we have not tagged ngx pagespeed alpine 3.4 version, as path needs to be updated once pagespeed official release happens. After 3.4 tag, we will tag 3.6 version with updated dockerfile. So as of now we can keep same path.